ephios/plugins/basesignup/templates/basesignup/participation_card_inline.html

```diff
@@ -26,8 +26,8 @@
         {% if participation.participant.is_minor %}
             <span class="badge bg-warning">{% trans "Minor" %}</span>
         {% endif %}
-        {% for qualification in participation.participant.qualifications|get_relevant_qualifications %}
-            <span class='badge bg-secondary text-dark'>{{ qualification }}</span>
+        {% for abbreviation in participation.participant.qualifications|qualifications_to_essential_abbreviations %}
```

Comment:
I was confused because the "only those that should be shown with user" is hidden in the template tag. Maybe you can include that in the name?

Reply:
I had that name with `essential` before I built the function filtering the "top level" qualifications which had "essential" in it as well. 

We could rename them to 

* `top_level` for removing qualifications included in others
* `to_show_with_user` for qualifications of categories to show with user 

which would make the thing be named `qualification_to_top_lebel_abbreviations_to_show_with_user`, but remember that's a different algorithm of itself. So maybe we should only rename the top level stuff and keep essential for the combination of both?!

Reply:
yeah sounds good. maybe define those terms in a docstring?

Reply:
Its now explained in `qualifications.py`